server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -260,6 +265,11 @@ it is around the label */
     display: none;
   }
 
+  // Show the government name when the logo is hidden
+  .cf-hidden-gov-name {
```

Comment:
Should this be `.cf-gov-name` also?

Reply:
Me forgetting to change that made me realize that we don't need this, only the other one :) but good catch!